source/documentation/Get started/quickstart.md

```diff
@@ -0,0 +1,210 @@
+
+- your account uses the free plan subscription
+- you choose a username that does not contain upper-case characters (for good practice)
+- you set your Git username for **every** repository on your device; see the [GitHub documentation](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git) for instructions
+- you follow the [Ministry of Justice's best practice guidelines](https://security-guidance.service.justice.gov.uk/passwords/#passwords) when setting your password
+- you [configure two-factor authentication (2FA)](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) on your mobile device; we recommend using either Google Authenticator or Microsoft Authenticator for 2FA
+    - For more information on 2FA within the Ministry of Justice, see the [MoJ's Security Guidance](https://security-guidance.service.justice.gov.uk/multi-factor-authentication-mfa-guide/#multi-factor-authentication-mfa-guide)
+
+## 4. Access the Analytical Platform
+
+Once you have your GitHub account, there are **two more steps** to complete before you can access the Analytical Platform: 
+
+- joining the MoJ Analytical Services GitHub organisation 
+- signing in to the Analytical Platform's Control Panel
+
+### Join MoJ Analytical Services
+
+After configuring your GitHub account you can request access to the Analytical Platform.
+
+Navigate to the [MoJ Analytical Services organisation](https://github.com/moj-analytical-services) and request to join it. The Operations Engineering team will review the request. If they approve your request, you will receive an email with a link to accept the invite.
+
+If you do not receive a response within 24 hours, request access in either the [**#ask-operations-engineering**](https://mojdt.slack.com/archives/C01BUKJSZD4) Slack channel or email **operations-engineering@digital.justice.gov.uk**, providing your GitHub username in your message.
+
+### Sign in to the Control Panel
+
+The main entry point to the Analytical Platform is the [Control Panel](https://controlpanel.services.analytical-platform.service.justice.gov.uk/). From there, you configure core tools such as JupyterLab and RStudio.
+
+When you access the Control Panel for the first time, a prompt will appear requiring you to configure 2FA using your mobile device. Note that while you use your GitHub account to access the Control Panel, **this 2FA is separate from the one you use to log in to GitHub**. You may need to disable browser extensions such as Dark Mode during the 2FA setup process.
+
+After you log in to the Control Panel for the first time, you can begin requesting access to data on the platform.
+
```

Comment:
Looks good up until this point, but after that why are we directing people to download and install JupyterLab and RStudio tools on their local machines? They're available on the AP and ready to use.

Reply:
Will fix